src/Admin/Pool.php

```diff
@@ -197,6 +197,8 @@ public function getAdminsByGroup($group)
      * @param string $class
      *
      * @return AdminInterface|null
+     *
+     * @phpstan-param class-string $class
```

Comment:
```
@phpstan-template T
@phpstan-param class-string<T> $class
@phpstan-return AdminInterface<T>|null
```

Reply:
phpstan got some troubles to resolve this correctly. So I added this to the baseline